docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md

```diff
@@ -0,0 +1,61 @@
+---
+id: using-aws-managed-opensearch
+title: "Using AWS Managed Opensearch"
+description: "Learn how to use an AWS managed opensearch instance in Camunda 8 Self-Managed deployment."
+---
+
+Camunda 8 Self-Managed has two different types of components: Camunda components (Operate, Optimize, Tasklist, etc.) and non-Camunda, dependency components (such as Keycloak and Elasticsearch). For more details, review the [architecture](../../../platform-architecture/overview.md) documentation for more information on the different types of applications.
+
+This guide steps through using an existing AWS managed opensearch instance. By default, [Helm chart deployment](../deploy.md) creates a new elassticsearch instance, but it's possible to use AWS managed opensearch instead.
+
+## Preparation
+
+### Authentication
+
+There are two layers of permissions with opensearch. IAM and Opensearch internal. If you would like to connect to Opensearch using IAM roles for service accounts (IRSA) then please also refer to this [page](../platforms/amazon-eks/irsa.md)
```

Comment:
`AWS IAM`

Reply:
> please also refer to this [page](...)

For better readability, we should use a full title for the link instead of just `page`.

Reply:
Also, let's link to the `#opensearch` section directly since the page has other irrelevant content.

Reply:
- changed to AWS IAM
- added full title
- linked to the specific opensearch section